tracer/src/Datadog.Trace.Tools.dd_dotnet/Checks/ProcessConfiguration.cs

```diff
@@ -43,12 +43,16 @@ internal static IConfigurationSource ExtractConfigurationSource(this ProcessInfo
             }
         }
 
-        var jsonConfigurationSource = JsonConfigurationSource.TryLoadJsonConfigurationFile(configurationSource, baseDirectory);
-
+        var jsonConfigurationSource = JsonConfigurationSource.TryLoadJsonConfigurationFile(configurationSource, baseDirectory, out string jsonDirectory);
```

Comment:
nitpick/non-blocking just wondering

So this would output something like

`Loaded JSON configuration file from: C:\Program Files\Datadog\datadog.json`

or

`Did not find JSON configuration file to load from: C:\Program Files\Datadog\datadog.json`

One question I have is whether or not it'd be beneficial to know where it is being sourced from?
At the moment this `jsonDirectory` can come from `ConfigurationKeys.ConfigurationFileName`'s value, `DD_DOTNET_TRACER_CONFIG_FILE`'s value, or `Path.Combine(baseDirectory ?? AppDomain.CurrentDomain.BaseDirectory, "datadog.json")` value

Would it be beneficial to also know the source it is pulling from and not just the value?

So trying to make it emit something like:

`Did not find JSON configuration file to load from default path: C:\Program Files\Datadog\datadog.json`
or 
`Loaded JSON configuration file from DD_DOTNET_TRACER_CONFIG_FILE`

Thoughts?

Reply:
It's only not default unless we use the environment variables right? 
perhaps someone could unintentionally use `DD_DOTNET_TRACER_CONFIG_FILE` without knowing.. 

I think this is something we an easily assume via context of the directory? but it wouldn't hurt to also add it in. 
Looking into seeing if there's a pretty way to do this.

Reply:
talked to Steven on slack. Going to push working on printing out the source at a later time.